src/utils.js

```diff
@@ -67,6 +67,9 @@ const gatsbyVersionIsPrerelease = prerelease(gatsbyVersion);
 const shouldUpgradeGatsbyVersion =
   lt(gatsbyVersion, GATSBY_VERSION_MANIFEST_V2) && !gatsbyVersionIsPrerelease;
 
+// We only want to create one manifest for all locales, in the future we'll have a more robust solution for node manifests and locales.
+const nodeManifestsIdsCreatedSet = new Set();
```

Comment:
This is problematic because if the "wrong" language is previewed first it will be the default one the user will be routed to until the process exits. So if you have 2 languages for a node and the second has no page created for it but that second one is sourced first, the first one (which has a page created for it) will never be routed to.